src/bitcoin-tx.cpp

```diff
@@ -23,6 +25,8 @@
 
 using namespace std;
 
+CCoinsViewCache *pcoinsTip = nullptr;
```

Comment:
These were added to avoid undefined referenced in `zen-tx` when `pcoinsTip` was referenced in the sidechain compilation unit. Such changes are now superseded, so these lines were removed.